e2e/cypress/integration/messaging/focus_move_spec.js

```diff
@@ -7,6 +7,42 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
+function verifyFocusInAddChannelMemberModal() {
+    // # Click to access the Channel menu
+    cy.get('#channelHeaderTitle').click();
+
+    // * The dropdown menu of the channel header should be visible;
+    cy.get('#channelHeaderDropdownMenu').should('be.visible');
+
+    // # 1. Click 'Add Members'
```

Comment:
```suggestion
    // # Click 'Add Members'
```

Reply:
I've made the changes and updated to master. Let me know if any other changes need to be made :)

Reply:
New updates look good to me.